Makefile

```diff
@@ -132,6 +132,24 @@ docker-buildx: test ## Build and push docker image for the manager for cross-pla
 	- docker buildx rm project-v3-builder
 	rm Dockerfile.cross
 
+###########
+# Release #
+###########
+
+##@ release:
+
+export DISABLE_RELEASE_PIPELINE ?= true
+substitute:
+	envsubst < .goreleaser.template.yml > .goreleaser.yml
```

Comment:
Is this used anywhere other than `release`? If not, I'd propose to consolidate this into the `release` recipe.

Having a top-level target called `substitute` seems like it could be both too vague and potentially confusing for contributors wondering if/when they should use it.